low-level/files/opt/seagate/sspl/bin/manifest_support_bundle

```diff
@@ -73,15 +75,15 @@ class ManifestSupportBundle(object):
                 # Capturing enclosure data only when sspl state is active
                 sspl_state = sspl_state[1] if len(sspl_state)==2 else ''
                 if sspl_state == 'active':
-                    subprocess.call([f"/opt/seagate/{PRODUCT_FAMILY}/sspl/lib/resource_health_view", "-n", "-e", "-s"], shell=False)
+                    subprocess.call([f"/usr/bin/resource_health_view", "-n", "-e", "-s"], shell=False)
```

Comment:
All these paths should be defined in a constant & used

Reply:
Done